Doc/library/ctypes.rst

```diff
@@ -2741,3 +2739,32 @@ Arrays and pointers
 
         Returns the object to which to pointer points.  Assigning to this
         attribute changes the pointer to point to the assigned object.
+
+
+.. _ctypes-exceptions:
+
+Exceptions
+^^^^^^^^^^
+
+.. exception:: ArgumentError
+
+   This exception is raised when a foreign function call cannot convert one of the
+   passed arguments.
+
+
+.. exception:: COMError(hresult, text, details)
+
+   Windows only: This non-public exception is raised when a COM method call
+   failed.
+
+   .. attribute:: hresult
+
+      The integer value representing the error code.
+
+   .. attribute:: text
+
+      The error message.
+
+   .. attribute:: details
+
+      The 5-tuple representing additional details about the error.
```

Comment:
That's not very helpful. Could you document what the items are?

Reply:
Since `_ctypes` is internal API that is subject to change, I think it would be best to change `test_win32` and `comtypes` so that they raise the same 5-tuple as `ctypes` itself, and document that 5-tuple.